data/schema-config.yaml

```diff
@@ -1535,6 +1535,8 @@ gene to pathway association:
     from: gene
     to: pathway
   properties:
+    name: str
+    inverse_name: str
```

Comment:
this adapter uses biocypher, so we need to add to the schema.

Other adapters won't need it because we don't use `name` and `inverse_name` in the API.